src/routes/expenses-routes.js

```diff
@@ -0,0 +1,21 @@
+const router = require('express').Router();
+const express = require('express');
+const {
+  getExpenses,
+  getExpenseById,
+  postExpense,
+  deleteExpense,
+  updateExpense,
+} = require('../controllers/expenses-controllers');
+
+router.get('/expenses', express.json(), getExpenses);
+
+router.post('/expenses', express.json(), postExpense);
+
+router.get('/expenses/:id', getExpenseById);
+
+router.delete('/expenses/:id', deleteExpense);
+
+router.patch('/expenses/:id', express.json(), updateExpense);
```

Comment:
```suggestion
const express = require('express');
const {
  getExpenses,
  getExpenseById,
  postExpense,
  deleteExpense,
  updateExpense,
} = require('../controllers/expenses-controllers');

const router = express.Router();

router.get('/', getExpenses);

router.post('/', postExpense);

router.get('/:id', getExpenseById);

router.delete('/:id', deleteExpense);

router.patch('/:id', updateExpense);
```
- Don't require express two time
- Specify base route in create server function `app.use('/expenses', expensesRoutes);
- Middleware with express.json you can specify only once too `app.use(express.json())`